index.html

```diff
@@ -181,6 +191,48 @@ <h2 class="workheading">Multi-Post Stories</h2>
             <button type="button" class="projectButton">See Project</button>
         </div>
 
+        <footer class="footer-form">
+            <div class="footer-heading">
+                <h2 class="contact-heading">CONTACT US</h2>
+                <p class="contact-details">
+                    If you have an application you are<br>
+                    interested in developing, a feature that you<br>
+                    need built or a project that needs coding.<br>
+                    I’d love to help with it.
+                </p>
+            </div>
+            <form action="https://formspree.io/f/mqkjaegz" method="post">
+                <ul class="footer-input">
+                    <li>
+                        <input name="Fullname"  class="item1" type="text" maxlength="30" placeholder="Full name" required>
+                    </li>
+                    <li>
+                        <input name="Email" class="item2" type="email" placeholder="Email address" required>
+                    </li>
+
+                    <li>
+                        <textarea name="message" class="item3" required maxlength="500" placeholder="Enter text here"></textarea>
+                    </li>
+                    
+                </ul>
+                <div class="footerButtonBox">
+                    <button type="submit" class="footerButton">See Project</button>
```

Comment:
- [ OPTIONAL ] According to the requirements kindly Implement the button interactions (enable, hover, etc.). because that very important to make your website more reactive 💯🚀🚀🚀